src/routes.json

```diff
@@ -205,7 +205,7 @@
         "title": "Scratch 1.4"
     },
     {
-        "name": "scratch2",
+        "name": "download-scratch2",
```

Comment:
is this renaming just to be clearer? Or does it have some functional effect?

Reply:
This really has a functional effect. The `make translations` script iterates over the non-redirect routes in the `routes.json` to build a translations file for each view. The `name` of the route is what the script uses to look up the translation files. So if the name doesn't match, you get the 'no translations for ...' for every language.